sbin/newfs_msdos/newfs_msdos.8

```diff
@@ -224,6 +224,12 @@ Create a file system, using default parameters, on
 newfs_msdos /dev/ada0s1
 .Ed
 .Pp
+Create a FAT32 filesystem with a 32kb allocation size on
```

Comment:
```suggestion
Create a FAT32 filesystem with a 32K allocation size on
```

This is consistent with other places in this manual page (in particular, the description of `-c`)